apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -432,9 +441,22 @@ impl Telemetry {
                     .map(|s| s.as_str())
                     .unwrap_or("info");
 
-                let sub_builder = tracing_subscriber::fmt::fmt().with_env_filter(
-                    EnvFilter::try_new(log_level).context("could not parse log configuration")?,
-                );
+                let formatter = debug_fn(|writer, field, value| {
+                    if field.name().starts_with("apollo_private") {
+                        // XXX: Are these being correctly delimited?
+                        write!(writer, "{}: null", field)
+                    } else {
+                        write!(writer, "{}: {:?}", field, value)
```

Comment:
Looks great, just a nit, in the previous formatted version we had fields displayed like this `myfield=content myotherfield=content2` should we try to copy that behavior or not ? Also fields name are in italic but that’s not very important in that case I think